help/access/file-shares.md

```diff
@@ -1,302 +1,237 @@
 
-   > Tip: Use built-in [smart search](../datagrok/smart-search.md) to find files and folders of interest quickly. You can
-   > type a word or part of a word to search for the following:
-   >
-   >* File and folder names
-   >* File extension
-   >* File and folder content metadata.<!--is this a complete list?-->
+![Suggest an application based on file types](clinical-case-file-manager.gif)
 
-Datagrok has multiple tools to help you find information you need precisely when you need it.
+</details>
+<details>
+<summary> Example: Create custom cell renderers </summary>
 
-* A [Treemap](../visualize/viewers/tree-map.md) helps you visually profile the content of your current folder and identify
-  the files that take up
-  the most space.
-  > Developers: You can [create custom folder viewers](../develop/how-to/folder-content-preview.md).
-  >
-  ><!--gif clinical data-->
-  >
-  >In the example above, a script is invoked against the folder content. If the folder contains files matching the file
-  > extension parameter, the **Preview** shows a custom widget (in this case - the application launch button) every time
-  > the folder is opened.
+In this example, a [script](/develop/how-to/custom-cell-renderers.md) is executed against the [SMILES](https://en.wikipedia.org/wiki/Simplified_molecular-input_line-entry_system) strings within the CSV file. The script computes the structure graph and 2D positional data, and renders the structure graphically.
 
-* You can browse and preview the content of individual files in [30+ formats](supported-formats.md). What you see is
-  dynamically adjusted based on the file type. For example:
+![Smiles renderer](Smiles-renderer.png)
 
-  * You can view the content of ZIP files just like regular folders.
-  * View and edit Markdown for HTML files.
-  * View image data in cell output and as files preview.
-  * Scroll through an interactive spreadsheet for CSV files.
-  * Look at protein structure from multiple angles using a 3D protein browser.
+</details>
 
-   <!--gif-->
+:::
 
-  > Developers: You can add custom formats via [package extensions](../develop/how-to/create-package.md).
-  > Note: File preview is limited to files less than 10MB in size. The platform won't display bigger files.
-  >
-  >You can't preview unsupported file formats, but you can download these files.
+Lastly, the [Context Panel](../datagrok/navigation.md#properties) provides specific information about a selected object and available actions. It works with folders, files, and data contained within these files. For example, when you click a CSV file, the **Context Panel** updates to show the file's metadata, available context actions, and other relevant information. If you subsequently click any of the dataframe's columns in the **Preview**, the **Context Panel** will update to display information and actions specific to that column, such as summary statistics for the column under **Stats**, or its data and semantic types under **Details**.
```

Comment:
Suggest to drop "Lastly" as unnecessary and hard to refactor :)

Reply:
Lastly, the [Context Panel](../datagrok/navigation.md#properties) provides specific information about a selected object and available actions
=>
Context Panel provides additional information about a selected file or folder, and the ability to execute contextual actions.

Now we can drop the following sentence